probe-rs/src/architecture/arm/core/mod.rs

```diff
@@ -286,7 +294,234 @@ static ARM_REGISTER_FILE: RegisterFile = RegisterFile {
     psp: Some(&register::PSP),
     extra: Some(&register::EXTRA),
     psr: Some(&register::XPSR),
-    // TODO: Floating point support
+
+    fpscr: Some(&register::FPSCR),
+    fpu_registers: Some(&[
```

Comment:
Total nit but I think I prefer `fp_registers` to `fpu_registers`.  Some SoCs don't have a dedicated unit for floating point.